docs/user-guide/options.md

```diff
@@ -259,4 +259,9 @@ CLI flag: `--quiet-deprecation-warnings`
 
 Ignore deprecation warnings.
 
+> [!TIP]
+> For Node.js 20.11.0+, you can disable individual deprecation warnings using the Node.js [`--disable-warning`](https://nodejs.org/api/cli.html#--disable-warningcode-or-type) mechanism, e.g.:
+>
+> `NODE_OPTIONS='--disable-warning=stylelint:005' npx stylelint "**/*.css"`
```

Comment:
[suggestion] How about converting it to a block like this? It can be easily copied via the button and supports syntax highlighting.

```suggestion
> ```shell
> NODE_OPTIONS='--disable-warning=stylelint:005' npx stylelint "**/*.css"
> ```
```